### ISSUE TITLE
Fix initialization for abacus_last_known_message_leaf_index

### ISSUE THREAD

**Issue by mattiecnvr** [abacus-network/abacus-monorepo#650]
The gauge is currently not initialized on startup and this is creating wonky metrics where it looks like thousands of messages are processed all at once.

**Comment by mattiecnvr** [comment]
Back burning this, might be resolved with the incoming changes on #637

**Comment by nambrot** [comment]
IIUC, both index-based and counter-based metrics are still useful cc @webbhorn ?

**Comment by mattiecnvr** [comment]
> IIUC, both index-based and counter-based metrics are still useful

Yes, both are useful. Though for most of our charts the count is more appropriate.
